assets/css/v2/style.css

```
@@ -997,6 +997,14 @@ blockquote p:last-child {
   }
 }
 
+/* Images */
+.figure-bitmap {
+  border: solid 2px #42932d;
+  margin-bottom: 10px;
```

```suggestion
  margin-bottom: 0.625rem;
```
Eventually we may want to re-evaluate if this is the place where we want to set padding.